### PR TITLE
Ks/patch sizes bug

### PR DESCRIPTION
**Pull request by thekevinscott**

### CODE REVIEW

**Comment by danwexler** [comment]
Aside from the version bumps, it looks like you changed the default idealo/gans model to 87? Can you provide more details about what changed? Also, why switch to hardcoding the model number instead of using `:latest`?

File: packages/upscalerjs/src/constants.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@
 This is an autogenerated file. See scripts/pin-cdn.js for more information.
 *****/
 
-export const ROOT = 'https://unpkg.com/@upscalerjs/models@0.8.26';
+export const ROOT = 'https://unpkg.com/@upscalerjs/models@0.8.27';
```
Comment (danwexler) [inline packages/upscalerjs/src/constants.ts:5]: In my old local version, `@latest` is added by utils/buildUrl. Why go to hard-coded versions?

**Comment by thekevinscott** [comment]
> Aside from the version bumps, it looks like you changed the default idealo/gans model to 27? Can you provide more details about what changed? Also, why switch to hardcoding the model number instead of using :latest?

Fair questions! I'll answer the second one first.

It's to avoid breaking previous versions of the upscaler if something changes in the model code. If `src/constants` refers to `@latest`, and we push a breaking change to the model's code, it means that any consumers will have to upgrade the upscaler package or have a broken installation. By pinning it to versions, `0.8.27` will always work even if `0.9.0` introduces a breaking change in the model.

That said, it raises other questions, like "what changed"? Nothing changed in the model code. For the upscaler, I'm working on fixing https://github.com/thekevinscott/UpscalerJS/issues/118 (though this is still an open issue).

Perhaps a better option would be for the version numbers of each package to drift independently; so, only when the model code is updated does this pinned version get changed. I'm not entirely sure how to do that in an automated way but I'm sure there's a way. But I'm also not sure if it's easier as a consumer to spot check that the versions match and move on.

Thoughts?

**Comment by danwexler** [comment]
Did the pre trained model change? Was it improved?

I have no objection to hardcoding the model number for the reasons you
mentioned. I just needed to hear them explained. The downside of your
approach is that you cannot change the model without changing the code, as
you have already noted.

I haven't looked closely enough at the idealo repo to see how they manage
their models. If they do their own versioning, and maintain a latest
branch, perhaps you could add another option to use their latest rather
than your fork? Or you could do that with your own repo - hard code the
default, and have a second option that loads latest?

**Comment by thekevinscott** [comment]
> Did the pre trained model change? Was it improved?

No, the model did not change at all.

> I haven't looked closely enough at the idealo repo to see how they manage
their models. If they do their own versioning, and maintain a latest
branch, perhaps you could add another option to use their latest rather
than your fork?

The idealo repo is Python-only, so it has to go through a compilation step to become Javascript, but we could rely on their versioning insofar as architecture is concerned.

However, what if we improve the model by, say, training it for another n epochs? That would presumably benefit from a version bump even though nothing's changed architecturally under the hood.

**Pull request merged**